pkgs/by-name/kb/kbd/package.nix

```
@@ -76,22 +79,25 @@ stdenv.mkDerivation rec {
   '';
 
   postInstall = ''
-    for i in $out/bin/unicode_{start,stop}; do
-      substituteInPlace "$i" \
-        --replace /usr/bin/tty ${coreutils}/bin/tty
+    for script in unicode_{start,stop}; do
+      substituteInPlace ''${!outputBin}/$script \
+        --replace-fail /usr/bin/tty ${coreutils}/bin/tty
+      moveToOutput "bin/$script" "$unicodeScripts"
```

This seems to be broken. The build fails for me with:

```
❯ nix-build -A pkgs.kbd
...
substitute(): ERROR: file '/nix/store/56rajyapplc5v18xq15487b2a6h6yz95-kbd-2.7.1/unicode_start' does not exist
...
```

This seems to work:

```suggestion
    for script in unicode_{start,stop}; do
      substituteInPlace ''${!outputBin}/bin/$script \
        --replace-fail /usr/bin/tty ${coreutils}/bin/tty
      moveToOutput "bin/$script" "$unicodeScripts"
```

---

Thanks for noticing! Fixed now.